### PR TITLE
Feat: user history

### PR DESCRIPTION
**Pull request by lemohammed**
Changeelog:

Added link to user page from title bar and added user created posts to /u/uid

### CODE REVIEW

File: src/Models/User.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default class User implements IUser {
     major: string;
     photoUrl?: string;
     ref: DocumentReference;
+    bio?: string;
```
Comment (RatikKapoor) [inline src/Models/User.ts:55]: 😍

**Review by RatikKapoor — APPROVED** [review]
🚀

**Review by rjb75 — APPROVED** [review]
✨

**Review by MrClean-1 — APPROVED** [review]
🔥 🚒 🧯

**Review by rjb75 — APPROVED** [review]
✨

**Review by MrClean-1 — APPROVED** [review]
🧇

**Pull request merged**